scripts/run.py

```diff
@@ -23,6 +23,10 @@
   --name NAME                    name for this teuthology run
   --suite-path SUITE_PATH        Location of ceph-qa-suite on disk. If not specified,
                                  it will be fetched
+  --interactive-on-error         drop to a python shell on failure, which will keep
+                                 job in halt state, developer can then ssh to
```

Comment:
"keep job in halt state" -> halt the job
comma should be semicolon

Reply:
updated.